lib/lotus/entity.rb

```diff
@@ -138,6 +138,17 @@ module ClassMethods
       #
       #   User.attributes => #<Set: {:id, :name}>
       #   DeletedUser.attributes => #<Set: {:id, :name, :deleted_at}>
+      #
+      # @example Add timestamps
+      #   require 'lotus/model'
+      #
+      #   class User
+      #     include Lotus::Entity
+      #     attributes :timestamps
```

Comment:
@joneslee85 separate method is good solution, you're right. 
But what do you think about dynamical creation of #created_at and #updated_at? After all, 3 places for declaration timestamps is a little uncomfortable.
Users can declare it in repository and in maps (inside collection). After an entity will be persisted and returned, they'll be have timestamps. 
Also I think about make them read-only for users. What do you think?